### PR TITLE
Add latest version commit hash to footer (notify-admin-204)

### PR DESCRIPTION
**Pull request by terrazoon**
## Description

Add the short commit hash to the footer.  Note that the only way I could test this in development was to put an environment variable in my .env file like:

COMMIT_HASH=hahahaha

the line in deploy.yml is supposed to retrieve the 7 digit short commit hash.

Screenshot 2024-02-26 at 11 10 54 AM

## Security Considerations

N/A

### CODE REVIEW

File: app/config.py [hunk 1]
```diff
@@ -36,6 +36,7 @@ class Config(object):
     NR_BROWSER_KEY = getenv("NR_BROWSER_KEY")
     settings = newrelic.agent.global_settings()
     NR_MONITOR_ON = settings and settings.monitor_mode
+    COMMIT_HASH = getenv("COMMIT_HASH", "Unknown")
```
Comment (ccostino) [inline app/config.py:39]: Let's change `Unknown` to several dashes instead:

```suggestion
    COMMIT_HASH = getenv("COMMIT_HASH", "-----")
```

**Review by ccostino — CHANGES_REQUESTED** [review]
Thanks, @terrazoon!  All looks good to me here, I have just one minor request for adjusting what we display in the case of noting having a commit hash available - suggestion already given! :-)

Speaking of which, I can't think of any situation where that wouldn't come through unless the `git` command failed for some reason, but even in that scenario I suspect that other failures would be taking place as well.  Is there any situation you can think of where just that command would fail and if so, is there a fallback we need in the script to account for it so the whole thing doesn't fail?

**Comment by terrazoon** [comment]
> Thanks, @terrazoon! All looks good to me here, I have just one minor request for adjusting what we display in the case of noting having a commit hash available - suggestion already given! :-)
> 
> Speaking of which, I can't think of any situation where that wouldn't come through unless the `git` command failed for some reason, but even in that scenario I suspect that other failures would be taking place as well. Is there any situation you can think of where just that command would fail and if so, is there a fallback we need in the script to account for it so the whole thing doesn't fail?

I think the dashes are enough.  Obviously we don't want it to fail so if we see dashes on staging, will have to debug there.

**Review by ccostino — APPROVED** [review]
Thanks, @terrazoon!

**Pull request merged**